lib/stat_tracker.rb

```diff
@@ -1,226 +1,89 @@
+require './lib/game.rb'
+require './lib/league.rb'
+require './lib/team.rb'
+require './lib/season.rb'
+require './spec/spec_helper'
 
 class StatTracker
-  attr_reader :games, :teams, :game_teams, :home_goals, :away_goals, :season
+  attr_reader :games, :teams, :game_teams, :game, :league, :season, :team
 
   def initialize(locations)
     @games = CSV.read locations[:games], headers: true, header_converters: :symbol
     @teams = CSV.read locations[:teams], headers: true, header_converters: :symbol
     @game_teams = CSV.read locations[:game_teams], headers: true, header_converters: :symbol
-    @home_goals = []
-    @away_goals = []
-    @season = []
+
+    @game = Game.new(games, teams, game_teams)
+    @league = League.new(games, teams, game_teams)
+    @season = Season.new(games, teams, game_teams)
+    @team = Team.new(games, teams, game_teams)
+    # @home_goals = []
+    # @away_goals = []
+    # @season = []
+    #@game = Game.new(locations)
```

Comment:
We will likely be deleted lines 19-22 -- need to chat with Jake before doing that.